templates/components/faceted-search/range-form.html

```diff
@@ -1,4 +1,4 @@
-<form id="facet-range-form" class="form" method="get" data-faceted-search-range novalidate>
+<form id="facet-range-form" class="form form--hiddenLabels" method="get" data-faceted-search-range novalidate>
```

Comment:
`.form--hiddenLabels` can affect only inner`.form-label`s, but in range form we do not use this class. So do we really need to add it?